doc/sphinx/source/notes_7_0.rst

```diff
@@ -2,10 +2,27 @@
 Release Notes for PoCL 6.1
 **************************
 
+* removed support for building tests & examples with OpenCL < 3.0
+
+* Support Khronos ICD if the version >= 3.0
```

Comment:
These look more internal/general notes. We should put feature highlights firt.

Reply:
I think Khronos ICD support is important for potential users, so IMO definitely not "internal" note, and isn't driver specific, hence added in this section.

I added **general** notes in this section (the driver-specific notes are in each driver's own section); there is no "release highlights" section in this document (with this PR). We can add one, but i think it makes more sense having "release highlights" in the Github release page, not in the detailed release notes (this file), but i don't have a strong opinion.

Reply:
[Seems in 6.0](https://portablecl.org/docs/html/notes_6_0.html) we had a sort of a "highlights" section in the beginning and after that driver-specific, and in the end notable-fixes and a deprecation list. I think it is clean and nice. Also we can copy paste the summary to the github release page easily then.

Reply:
> I think it is clean and nice
 
It's missing a section for user-related changes that are not "highlights" but still important to know. Like e.g. changing the minimum CMake version.

I added "new features" (highlights) and "notable user-facing changes" sections at the beginning.